torch/csrc/lazy/core/lazy_view.cpp

```diff
@@ -126,6 +133,12 @@ Value ApplyUpdate(Value ir_value, const Alias::UpdateData& update_data) {
             view_info.diagonal->dim1,
             view_info.diagonal->dim2);
         break;
+      case ViewInfo::Type::kExpand:
```

Comment:
it's not obvious to me if this is OK.

if someone first calls expand, then modifies the result, the 'view' system would track that modification and make sure any other users also get it.  in your way, directly returning an 'Expand' node here, does it lose the view tracking? or is this somehow interposing late enough that the view tracking already happened?

Reply:
It won't work. @Krovatkin is handling this op now.